source/src.n68equiv/model_specific.F90

```diff
@@ -35,52 +35,54 @@ subroutine setup_major_gas_matrix
 !------------------------------------------------------------------------
 ! PURPOSE:  Load full k-distribution from files into major gas absorber
 !           matrix.
-!------------------------------------------------------------------------ 
+!------------------------------------------------------------------------
 
     use kabs
     implicit none
 
-!------------------------------------------------------------------------ 
+!------------------------------------------------------------------------
 !
 ! Local Variables
 !
   integer :: iw
-!------------------------------------------------------------------------ 
+!------------------------------------------------------------------------
 ! Start Code
 !
 
     ! Initialize all bins to zero
     k_major_data(:,:,:,:,:) = 0.0
-    k_major_data(iH2O, :,:,:,:) = k_h2o(:,:,:,:) 
+    k_major_data(iH2O, :,:,:,:) = k_h2o(:,:,:,:)
     k_major_data(iCO2, :,:,:,:) = k_co2(:,:,:,:)
     k_major_data(iCH4, :,:,:,:) = k_ch4(:,:,:,:)
-    k_major_data(iC2H6,:,:,:,:) = k_ch4(:,:,:,:)
```

Comment:
ETW: note to self.  Russell identified an error where the C2H6 absorption coefficients were incorrectly being set to those of CH4.  No publications to date have used this gas.  I need to redo a few unpublished simulations.

Reply:
Oh man! I'm sorry, Eric, I totally forgot to tell you about this! I hope it doesn't take too much work or compute time to redo those...

Reply:
I am glad you caught that!  It's not much bother, I have a few unpublished simulations with C2H6 to restart.  Glad it got found.